tests/acceptance/features/apiSharingNg/enableDisableShareSync.feature

```diff
@@ -867,7 +867,7 @@ Feature:  enable or disable sync of incoming shares
       | sharee          | Brian      |
       | shareType       | user       |
       | permissionsRole | Viewer     |
-    When user "Brian" enables sync of share "<resource>" offered by "Alice" from "NewSpace" space using the Graph API
+    And user "Brian" enables sync of share "<resource>" offered by "Alice" from "NewSpace" space using the Graph API
```

Comment:
Note: the linter did what it could.
Actually, line 860 should be `Given` and this line 870 be `When`
That could be manually adjusted in a future commit.

Reply:
Okay, will update and push it ASAP!